lib/fencing/st_output.c

```diff
@@ -47,7 +47,7 @@ stonith__failed_history(pcmk__output_t *out, va_list args) {
             continue;
         }
 
-        if (!pcmk__str_in_list(only_node, hp->target)) {
+        if (!pcmk__str_in_list(only_node, hp->target, pcmk__str_casei)) {
```

Comment:
I would use only pcmk__str_none in this commit, then do a separate commit setting pcmk__str_casei as a bug fix commit

Reply:
Fixed.